server/settings/editorial_creators.py

```diff
@@ -7,20 +7,28 @@
 
 class ClipNameTokenizerItem(BaseSettingsModel):
     _layout = "compact"
-    name: str = SettingsField("", title="Tokenizer name")
-    regex: str = SettingsField("", title="Tokenizer regex")
+    name: str = SettingsField("", title="Token name")
+    regex: str = SettingsField("", title="Token regex")
 
 
 class ShotAddTasksItem(BaseSettingsModel):
     _layout = "compact"
-    name: str = SettingsField('', title="Key")
+    name: str = SettingsField('', title="Task Name")
```

Comment:
very very picky, just for my own OCD so it matches the others. 
```suggestion
    name: str = SettingsField('', title="Task name")
```